Assets/Scripts/Strategy/BaseManagement/Attachable/HomeBase.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class HomeBase : MonoBehaviour
+{
+    public BaseManager bm;
+
+    public Canvas canvasObject;
+    [Range (1, 3)]public int tier;
+
+    public Text tierDisplayed;
+
+    Renderer renderer;
+
+    private void Awake()
+    {
+        renderer = gameObject.GetComponent<Renderer>();
+    }
+
+    void Start()
+    {
+        tier = 1;
+        renderer.material.color = Color.gray;
+        tierDisplayed.text = tier.ToString();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.U))
+        {
+            UpgradeTier();
+        }
+    }
+
+    private void OnMouseOver()
+    {
+        Debug.Log("Enter");
+
+        renderer.material.color = Color.yellow;
+
+        if (Input.GetMouseButtonDown(0))
+        {
+            canvasObject.gameObject.SetActive(true);
+            Debug.Log("I was clicked.");
+            bm.GetComponent<BaseManager>().SetAllCanvasInactive();
+        } 
+    }
+
+    private void OnMouseExit()
+    {
+        Debug.Log("Exit");

```

Comment:
The debug messages should be removed. If you think they should stay it should be clear where they are coming from and not just say a single word

Reply:
e7eb51e